_includes/hero.html

```diff
@@ -1,59 +1,57 @@
+            <article class="hero_preview">
                 {% if related_post %}
-                    <span class="category-slug">
-                        {{ category_icon(related_post.category.0, 'category-slug_icon') }}
-                        <span class="u-visually-hidden">Category:</span>
-                        {{ related_post.category.0 }}
-                    </span>
+                <span class="category-slug">
+                    {{ category_icon(related_post.category.0, 'category-slug_icon') }}
+                    <span class="u-visually-hidden">Category:</span>
+                    {{- related_post.category.0 -}}
+                </span>
                 {% endif %}
-                    <div class="summary">
-                        <h1 class="summary_header">
-                            {{ hero.title }}
-                        </h1>
-                        <p class="summary_text">
-                        {% if hero.content %}
-                            {{ hero.content|striptags }}
-                        {% elif related_post %}
-                            {% if related_post.dek %}
-                                {{ related_post.dek|striptags }}
-                            {% else %}
-                                {{ related_post.excerpt|striptags }}
-                            {% endif %}
-                        {% endif %}
-                        </p>
-                    {% if related_post %}
-                        {% if related_post.author.0 %}
+                <div class="summary">
+                    <header class="summary_header">
+                        <h1 class="summary_heading">{{ hero.title }}</h1>
+                    </header>
+                    <p class="summary_text">
+                    {%- if hero.content %}
+                        {{- hero.content|striptags -}}
+                    {%- elif related_post %}
+                        {%- if related_post.dek %}
+                            {{- related_post.dek|striptags -}}
+                        {% else %}
+                            {{- related_post.excerpt|striptags -}}
+                        {% endif -%}
+                    {% endif -%}
+                    </p>
+                    <footer class="summary_footer">
+                    {%- if related_post and related_post.author.0 %}
                         <p class="summary_byline">
                             By {{ related_post.author.0 }}
-                            {% if related_post.author.1 %}
+                            {%- if related_post.author.1 %}
                             and {{ related_post.author.1 }}
-                            {% endif %}
+                            {% endif -%}
                         </p>
-                        {% endif %}
-                    {% endif %}
-                    </div>
+                    {% endif -%}
+                    </footer>
                 </div>
-            </div>
+            </article>
```

Comment:
It did, until I started working with the summary for the [blog/newsroom previews](https://github.com/cfpb/cfgov-refresh/blob/gh-pages/_includes/posts-paginated.html#L23) and realized there's an existing pattern.